languages/rust/reference/exercise-concepts/poker.md

```diff
@@ -0,0 +1,35 @@
+# Concepts Required for the Poker exercise
+
+There are many ways to potentially solve an exercise, but I'll be looking at the [canonical example](https://github.com/exercism/rust/blob/c0afe3855e0e71dc5429aa97657e074aa90a8e11/exercises/poker/example.rs) as my primary reference for what concepts are used.
+
+When referring to the list of concepts, we mean [this list](https://github.com/exercism/v3/blob/4f164e3751202087ff40c8321238d4c283a90fd0/languages/rust/reference/README.md)
+
+Note that for both concept and practice exercises, the list of prerequisite concepts should be as small as possible. The goal here is not to enumerate all concepts which may be used in all possible approaches to the problem; it is to enumerate the minimal set of knowledge with which a determined student could conceivably solve the problem. 
+
+## Required concepts
+### Existing
+
+- fundamentals
+- `String` vs `&str`
+- Ownership / Borrowing
+- `Option<T>`
+- `Vec<T>`

```

Comment:
```suggestion
- `Vec<T>`
- Traits
- Enums
```
These are a couple concepts I thought could be added from cursory reads of the canonical solution and other public solutions.